src/runbenchmark.jl

```diff
@@ -73,11 +73,23 @@ function maybe_sleep(x)
 end
 
 function benchmark_fun!(
-    f!::F, summarystat, C, A, B, sleep_time, force_belapsed = false, reference = nothing
+    f!::F,
+    summarystat,
+    C,
+    A,
+    B,
+    sleep_time,
+    force_belapsed,
+    reference,
+    comment::String, # `comment` is a place to put the library name, the dimensions of the matrices, etc.
 ) where {F}
     maybe_sleep(sleep_time)
     t0 = @elapsed f!(C, A, B)
-    isnothing(reference) || @assert C ≈ reference
+    if (reference !== nothing) && (!(C ≈ reference))
+        msg = "C is not approximately equal to reference"
```

Comment:
The reference is the first of the libraries passed to `runbenchmark`, so perhaps the order does matter -- we should trust that the reference is correct.
When we get this error, we know either the ref or `lib` is wrong.

Reply:
Not quite sure what you mean. Can you elaborate on this?

Reply:
Oh I see what you mean.

Reply:
Should we instead just always use `LinearAlgebra.mul!` to compute the reference C0?

Reply:
That could take potentially many extra seconds at larger sizes, and do we trust it more than BinaryBuilder OpenBLAS or MKL (or BLIS)?
The time issue would be major when it's integers, and `mul!` calls the generic matmul.
I guess it'd be nice to always know what the reference is, though.

Reply:
That's a good point. Hmmm.

Reply:
Let's keep the current behavior for now.